ext/intl/resourcebundle/resourcebundle.stub.php

```diff
@@ -4,13 +4,13 @@
 
 class ResourceBundle implements IteratorAggregate, Countable
 {
-    public function __construct(?string $locale, ?string $bundlename, bool $fallback = true) {}
+    public function __construct(?string $locale, ?string $path, bool $fallback = true) {}
```

Comment:
Hm, I'm not sure about `$path` here, as this also accepts `"ICUDATA"` and various `"ICUDATA-xyz"` at least.

Reply:
In that case... what about bundle? I know that it's also not perfect, because ResourceBundle objects are called like this. Do you see any better choice?

Reply:
Don't really have a great suggestion here, let's go with `$bundle`.